### PR TITLE
🎉 Add Order Risks + New attributes to Orders schema for Native Shopify connector

### LINKED ISSUE #3482: Additional attributes in shopify schema
## Tell us about the problem you're trying to solve
For shopify in orders stream we need more attributes. Even though the API supports schema for order.json inside tap-shopify is not having some attributes.

## Describe the solution you’d like
Inside base-singer -> singer_helper file where we generate final catalog can't we use source as that which has more attributes by default

## Describe the alternative you’ve considered or used
Ofcourse forking tap-shopify and then changing airbyte-shopify image and use that.

### PR DESCRIPTION
**Pull request by bazarnov**
## What
Resolving issue:
#3482 - Additional attributes in shopify schema

## How
1) Added Schema for Order Risks (order_risks.json)
2) Added stream class OrderRisks() to source.py
3) Re-defined `orders.json' schema to add all possible attributes from Orders objects
3) Bumped version of the image

## Pre-merge Checklist
Expand the checklist which is relevant for this PR. 

<details><summary> <strong> Connector checklist </strong> </summary>
<p>

- [x] Issue acceptance criteria met
- [x] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)
- [x] Secrets are annotated with `airbyte_secret` in output spec
- [x] Unit & integration tests added as appropriate (and are passing)
- [x] `/test connector=connectors/<name>` command as documented [here](https://docs.airbyte.io/contributing-to-airbyte/building-new-connector#updating-an-existing-connector) is passing. 
    * Community members can skip this, Airbyters will run this for you. 
- [x] Code reviews completed
- [x] Credentials added to Github CI if needed and not already present. [instructions for injecting secrets into CI](https://docs.airbyte.io/contributing-to-airbyte/building-new-connector#using-credentials-in-ci). 
- [x] Documentation updated 
    - [x] CHANGELOG.md
- [ ] Build is successful
- [x] Connector version bumped like described [here](https://docs.airbyte.io/contributing-to-airbyte/building-new-connector#updating-a-connector)
- [ ] New Connector version released on Dockerhub by running the `/publish` command described [here](https://docs.airbyte.io/contributing-to-airbyte/building-new-connector#updating-a-connector)
- [ ] No major blockers
- [ ] PR merged into master branch
- [ ] Follow up tickets have been created
- [ ] Associated tickets have been closed & stakeholders notified
</p>
</details>

### CODE REVIEW

File: airbyte-integrations/connectors/source-shopify/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Shopify Source CHANGELOG
+
+| Date | Released Version | Notes |
+| :--- | :--- | :--- |
+| `2021-06-07` | `0.1.3` | `Initial Python CDK Airbyte Source Shopify Release` |
+| `2021-06-08` | `0.1.4` | `Added Order Risks Stream` |
```
Comment (sherifnada) [inline airbyte-integrations/connectors/source-shopify/CHANGELOG.md:6]: can you place it in reverse order, so most recent should be on top

**Review by sherifnada — CHANGES_REQUESTED** [review]
@bazarnov I believe the linked issue is not asking for the order risks stream, but for more attributes on the orders stream itself. See the PR contributed by @harshithmullapudi : https://github.com/airbytehq/tap-shopify/pull/4 where it contains the required fields

you can also just go to the orders schema on shopify API and make sure all attributes are reflected

@harshithmullapudi please correct me if I'm wrong

File: airbyte-integrations/connectors/source-shopify/CHANGELOG.md (hunk 1 above)
Comment (harshithmullapudi) [reply airbyte-integrations/connectors/source-shopify/CHANGELOG.md:6]: yeah https://github.com/airbytehq/tap-shopify/pull/4 this is for adding more attributes to orders stream.

Comment (bazarnov) [reply airbyte-integrations/connectors/source-shopify/CHANGELOG.md:6]: There are 84 attributes across Orders object inside Shopify. All of them are available by default. This is the standard output from API Call, we are not limiting anything here unless we skipping some specific fields, due to `orders.json` schema. 
Additionally, there are: 

- Orders Refunds
- Orders Transactions
- Orders Risks 
- Order Drafts

Those could be fetched using the order_id, for each order_id separately, we do so.

If we missing any attribute of `order.json` schema itself, please verify it using the following documentation on the official site, for availability. [link](https://shopify.dev/docs/admin-api/rest/reference/orders/order#index-2021-04)

For now, the only attribute of Order Stream in general that is not covered is Order Drafts.

Comment (bazarnov) [reply airbyte-integrations/connectors/source-shopify/CHANGELOG.md:6]: Found some missing structures inside `order.json`, working to add them.

Comment (bazarnov) [reply airbyte-integrations/connectors/source-shopify/CHANGELOG.md:5]: Done.

**Comment by bazarnov** [comment]
> @bazarnov I believe the linked issue is not asking for the order risks stream, but for more attributes on the orders stream itself. See the PR contributed by @harshithmullapudi : [airbytehq/tap-shopify#4](https://github.com/airbytehq/tap-shopify/pull/4) where it contains the required fields
> 
> you can also just go to the orders schema on shopify API and make sure all attributes are reflected
> 
> @harshithmullapudi please correct me if I'm wrong

Confusing part here was to mention **"Risks"** right down bellow the **"Add new attributes to shopify orders schema".**  I hope now the expectations are met)

**Review by sherifnada — APPROVED** [review]

**Pull request merged**